### ISSUE TITLE
How to pass the Alternative Payment information?

### ISSUE THREAD

**Issue by fralau** [claudep/swiss-qr-bill#40]
I am refering to these lines of codes:

https://github.com/claudep/swiss-qr-bill/blob/14c0be0124c1f122b99dc43310a9ddf4e0c294a1/qrbill/bill.py#L268-L269

According to the _SIX Implementation Guidelines QR-Bill_, p. 29, the limitation of 140 characters refers to the sum of the two fields.

- Unstructured Message (Ustrd)
- Bill Information(StrdBkgInf).
 
So, this seems correct.

But then, how we should pass the last 2 fields AltPmt, in case of need (100 characters each), without triggering this error?

**Comment by claudep** [comment]
Salut Laurent,

The `extra_infos` parameter is strictly intented for the indication zone, displayed between creditor and debtor.

I think we did not implement the "Alternate payment" zone yet. Are you interested to contribute that functionality?

**Comment by fralau** [comment]
OK, good to know.
🤔 For contributing the alternate payments: why not? (Providing I manage to find my way around?)

**Comment by fralau** [comment]
I put together a first solution (https://github.com/fralau/swiss-qr-bill) which now includes the full trailer.

To give an idea, here is the command line (let me know if this works for you?):

```bash
rbill --account CH4431999123000889012 --creditor-name "John Doe" --creditor-postalcode 1200 --creditor-city Genève --reference-number 210000000003139471430009017 --extra-info "Paying ..." --amount 358.60 --billing-info "//S1/10/10201409/11/200701/20/140.000- 53/30/102673831/31/200615/32/7.7/33/7.7:139.40/ 40/0:30" --alt-scheme "eBill/B/peter@sample.ch" "Name AV2: XY;XYService;54321"
```

This is the text output (I added a `--text` option):

```
SPC
0200
1
CH4431999123000889012
S
John Doe


1200
Genève
CH







358.60
CHF







QRR
210000000003139471430009017
Paying ...
EPD
//S1/10/10201409/11/200701/20/140.000- 53/30/102673831/31/200615/32/7.7/33/7.7:139.40/ 40/0:30
eBill/B/peter@sample.ch
Name AV2: XY;XYService;54321
```

Here is the result:
![CH4431999123000889012-2020-08-18_124512](https://user-images.githubusercontent.com/36767124/90504084-cfe8df80-e150-11ea-879a-b9ce64af4477.png)

The QR is correct, but the bill itself needs to be corrected. Here is the proper output, produced with the online simulation tool (https://www.qr-rechnung.net):

![QR-facture-1](https://user-images.githubusercontent.com/36767124/90504373-41289280-e151-11ea-88b6-3ba48bb0be94.png)

**Note**: I ran the test suite, according to it no bills were harmed in the process.

**Comment by claudep** [comment]
Your clone seems even with this one, did you forget to push your code change? A pull request would be nice, even if not finished.

**Comment by fralau** [comment]
Done: #41

**Comment by claudep** [comment]
Fixed by 05f487b643ad087ac5ea7e8d1e83a2424f339a9c
